tests/Unit/Context/Propagation/EnvironmentGetterSetterTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Unit\Context\Propagation;
+
+use InvalidArgumentException;
+use OpenTelemetry\Context\Propagation\EnvironmentGetterSetter;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(EnvironmentGetterSetter::class)]
+class EnvironmentGetterSetterTest extends TestCase
+{
+    protected function tearDown(): void
+    {
+        putenv('A');
```

Comment:
we have `OpenTelemetry\Tests\TestState` trait to handle setting and cleaning up the environment.